lib/lang.ts

```diff
@@ -66,10 +71,14 @@ const getTextValue = lookupDotNotation => {
     return missingTextData;
   }
 
+  if (typeof textValue !== 'string') {
+    return missingTextData;
+  }
+
   return textValue;
-};
+}
 
-const i18n = (lookupDotNotation, options = {}) => {
+export function i18n(lookupDotNotation: string, options = {}): string {
```

Comment:
`options` here should have a type, see https://github.com/HubSpot/hubspot-local-dev-lib/blob/main/utils/lang.ts#L111